checkov/common/runners/object_runner.py

```diff
@@ -138,3 +159,34 @@ def _change_files_path_to_relative(report: Report) -> None:
         for record in report.get_all_records():
             record.file_path = record.file_path.replace(os.getcwd(), "")
             record.resource = record.resource.replace(os.getcwd(), "")
+
+    def _get_triggers(self, result: Tuple[Dict[str, Any], Dict[str, Any]]) -> set[str]:
+        triggers_set = set()
+        triggers = result[0].get(True)
+        try:
+            if triggers:
+                # for key in triggers.keys():
+                #     if key != START_LINE and key != END_LINE:
+                #         triggers_set.add(key)
+                triggers_set = {key for key in triggers.keys() if key != START_LINE and key != END_LINE}
+
+        except Exception as e:
+            logging.info(f"error : {str(e)}")
+        return triggers_set
+
+    def _get_jobs(self, result: Tuple[Dict[str, Any], Dict[str, Any]]) -> dict[str, dict[str, str]]:
+        jobs_dict: dict[str, dict[str, str]] = {}
+        tmp_key: str = ""
+        jobs = result[0].get('jobs')
+        if jobs:
+            for key, value in jobs.items():
+                if key != START_LINE and key != END_LINE:
+                    jobs_dict[key] = {}
+                    tmp_key = key
+                else:
+                    if key == START_LINE:
+                        jobs_dict[tmp_key][START_LINE] = value
+                    if key == END_LINE:
+                        jobs_dict[tmp_key][END_LINE] = value
```

Comment:
A question regarding the usage of `tmp_key`.
In case we get to this block first, `tmp_key` is an empty string, also if we get to this block in a later iteration then `tmp_key` holds the last key that didn't get to this block - is that intended? if so, I suggest changing the variable name to `prev_key`

Reply:
changed it :)